cli/src/funTest/kotlin/OrtMainFunTest.kt

```diff
@@ -72,7 +72,7 @@ class OrtMainFunTest : StringSpec() {
                     packageCurationProviders = listOf(
                         PackageCurationProviderConfiguration(
                             type = "File",
-                            config = mapOf("path" to projectDir.resolve("gradle/curations.yml").path)
+                            config = mapOf("path" to "src/funTest/assets/gradle-curations.yml")
```

Comment:
consider: `getAssetFile("gradle-curations.yml").path`

Reply:
Done in https://github.com/oss-review-toolkit/ort/pull/6654 as this already got merged.